sw/device/silicon_creator/mask_rom/e2e/BUILD

```diff
@@ -103,3 +110,39 @@ opentitan_functest(
         "//sw/device/lib/testing/test_framework:ottf_main",
     ],
 )
+
+opentitan_functest(
+    name = "e2e_chip_specific_startup",
+    srcs = ["chip_specific_startup.c"],
+    args = [],
+    cw310 = cw310_params(
+        args = [
+            "--bitstream=\"$(location //hw/bitstream:mask_rom)\"",
+            "--rom-kind=maskrom",
+            "--bootstrap=\"$(location {flash})\"",
+        ],
+        bitstream = "//hw/bitstream:mask_rom",
+    ),
+    manifest = "//sw/device/silicon_creator/rom_ext:manifest_standard",
+    signed = True,
+    test_harness = "//sw/host/tests/mask_rom/e2e_chip_specific_startup",
+    verilator = verilator_params(
+        exit_failure = BOOT_FAILURE_MSG,
+        exit_success = BOOT_SUCCESS_MSG,
+        rom = "//sw/device/silicon_creator/mask_rom",
+        # Note: Leaving as broken since this test takes a long time and
+        # we don't really need it as long as the CI runs the FPGA version.
+        tags = [
+            "broken",
+        ],
```

Comment:
this probably isn't needed b/c it [looks like](https://cs.opensource.google/opentitan/opentitan/+/master:ci/scripts/run-verilator-tests.sh;l=1?q=run-verilator&ss=opentitan%2Fopentitan) we do not run said tests in CI anyway?

Reply:
Removed